grails-app/migrations/0.9.x/changelog-2023-09-07-1140-add-missing-is_root-value-to-root-catalog.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9 http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+
+  <changeSet author="drodzewicz" id="070920231140-0">
+    <preConditions onFail="MARK_RAN">
+        <sqlCheck expectedResult="1">
```

Comment:
I think it would be better if this checked for `WHERE id = 'ROOT' and is_root is NULL` and the expected result being 0.

Reply:
I added an additional WHERE condition.
As for the expected result 0 I disagree.
The precondition is set to `onFail="MARK_RAN"` which will skip this changelog if category id = ROOT and is_root = NULL won't be found, which is what I expect.